src/shogun/regression/KernelRidgeRegression.h

```diff
@@ -132,20 +119,23 @@ class CKernelRidgeRegression : public CKernelMachine
 		 */
 		virtual bool train_machine(CFeatures* data=NULL);
 
-	private:
-		void init();
-
-		/** train regression using Gauss-Seidel iterative method
+		/** Train regression using Cholesky decomposition
 		 *
-		 * @return whether training was successful
+		 * @param alphas vector of alpha already initialized
```

Comment:
BTW and for future comments: the vector here it is not initialized, but pre-allocated